src/test/java/com/ge/predix/uaa/token/lib/ZacTokenServiceTest.java

```diff
@@ -143,55 +160,52 @@ public void testDefaultAndZoneSpecificResourceAuthorization(final String zoneId,
     }
 
     private OAuth2Authentication loadAuthentication(String configuredHeaderNames, String configuredBaseDomains,
-            final String requestZoneName, final String userScopes) {
-        return loadAuthentication(configuredHeaderNames, configuredBaseDomains, requestZoneName, userScopes,
-                "/test/resource", Arrays.asList("/zone/**"));
+            final String requestZoneName, final String userScopes, Boolean useSubdomainsForZones) {
+        return loadAuthenticationWithZoneAsHeader(configuredHeaderNames, configuredBaseDomains, requestZoneName,
+                useSubdomainsForZones, userScopes, "/test/resource", Arrays.asList("/zone/**"));
```

Comment:
To improve readability, remove loadAuthentication and call loadAuthenticationWithZoneAsHeader explicitly.